src/BooleanParser.php

```diff
@@ -4,6 +4,11 @@
 
 class BooleanParser extends AbstractValueParser
 {
+    protected function describe()
+    {
+        return "either true or false";
```

Comment:
After more thoughts I would just keep it how it is because that's more simple :)